api4/channel.go

```diff
@@ -753,7 +753,7 @@ func getChannelsForTeamForUser(c *Context, w http.ResponseWriter, r *http.Reques
 		return
 	}
 
-	channels, err := c.App.GetChannelsForUser(c.Params.TeamId, c.Params.UserId, false)
+	channels, err := c.App.GetChannelsForUser(c.Params.TeamId, c.Params.UserId, c.Params.IncludeDeleted)
```

Comment:
is there a special permission that we need to check here to see if users are allowed to view deleted channels?

Reply:
AFAIK there is no special permission for archived channels. Taking a look at the current endpoint methods no one is asking for special permission